doc/sphinx-guides/source/api/external-tools.rst

```diff
@@ -70,19 +78,19 @@ Terminology
+    external tool manifest       A **JSON file** the defines the URL constructed by Dataverse when users click explore or configure tool options. External tool makers are asked to host this JSON file on a website (no app store yet, sorry) and explain how to use install and use the tool. Examples include :download:`fabulousFileTool.json <../_static/installation/files/root/external-tools/fabulousFileTool.json>` and :download:`dynamicDatasetTool.json <../_static/installation/files/root/external-tools/dynamicDatasetTool.json>` as well as the real world examples above such as Data Explorer.
 
     displayName                  The **name** of the tool in the Dataverse web interface. For example, "Data Explorer".
 
     description                  The **description** of the tool, which appears in a popup (for configure tools only) so the user who clicked the tool can learn about the tool before being redirected the tool in a new tab in their browser. HTML is supported.
 
     scope                        Whether the external tool appears and operates at the **file** level or the **dataset** level. Note that a file level tool much also specify the type of file it operates on (see "contentType" below).
 
-    type                         Whether the external tool is an **explore** tool or a **configure** tool. Configure tools require an API token because they make changes to data files (files within datasets). Configure tools are currently not supported at the dataset level (no "Configure" button appears in the GUI for datasets).
+    type                         Whether the external tool is an **explore** tool, a **preview** tool or a **configure** tool. Configure tools require an API token because they make changes to data files (files within datasets). Configure tools are currently not supported at the dataset level.
 
     toolUrl                      The **base URL** of the tool before query parameters are added.
     
-    hasPreviewMode               A boolean that indicates whether tool has a preview mode which can be embedded in the File Page. Since this view is designed for embedding within Dataverse, the preview mode for a tool will typically be a view without headers or other options that may be included with a tool that is designed to be launched in a new window. Sometimes, a tool will exist solely to preview files in Dataverse and the preview mode will be the same as the regular view. 
+    hasPreviewMode               A boolean that indicates whether tool has a preview mode which can be embedded in the File Page. Since this view is designed for embedding within Dataverse, the preview mode for a tool will typically be a view without headers or other options that may be included with a tool that is designed to be launched in a new window. 
 
```

Comment:
Do we want/need to say that hasPreviewMode only pertains to explore tools?

Reply:
As we discussed with @scolapasta we might do away with hasPreviewMode entirely and factor out "type" to its own table. The idea would be the perhaps tools could have multiple types, so an explore tool would also be a preview tool if it supports preview. We plan to discuss more next week.

Reply:
FWIW: Looking at 5.x with QDR, some of the previewers like audio are small (regardless of file size) and it really makes no sense to have a separate explore button on the file page, so being able to mark such a previewer as preview only would help there, and would make it easy for admins to decide whether other previewers are worth popping out as explore tools or not. So - support for allowing tools to register with one or more types.

Reply:
@qqmyers thanks. To be clear, even with the code as-is in this pull request, it's possible for admins to decide if a tool that offers preview appears under "explore" or not. To convert the audio tool to "preview only" (again, given the code so far), you'd change it from an "explore" tool to a "preview" tool. To switch it back, you'd change it from "preview" to "explore" and make sure hasPreviewMode is true.

Also, please note that from the Preview tab, there's always a button to open the tool in a new window. For preview tools the button is labeled "Open in a New Window" and for explore tools it's labeled as it was before, "Explore on [TOOL NAME]". So popping out happens for both.

Reply:
hasPreviewMode has been removed in 767882b